storage/ops.go

```diff
@@ -461,32 +488,28 @@ func swapMakeFsCommand(bd *BlockDevice, args []string) ([]string, error) {
 	return cmd, nil
 }
 
-func swapMakePartCommand(bd *BlockDevice, start uint64, end uint64) (string, error) {
+func swapMakePartCommand(bd *BlockDevice) (string, error) {
 	partName := "linux-swap"
 
 	if bd.FsType == "swap" && bd.Type == BlockDeviceTypeCrypt {
-		mapped := fmt.Sprintf("eswap%d", start)
+		mapped := fmt.Sprintf("eswap-%s", bd.Name)
```

Comment:
Was this a bug fix?

Reply:
start is no longer available to make the eswap mapped name unique, so switching to using the device name.

Reply:
Here is what the change will do to the swap name:
```
# lsblk
NAME                                        MAJ:MIN RM  SIZE RO TYPE  MOUNTPOINT
sr0                                          11:0    1 1024M  0 rom
vda                                         253:0    0    8G  0 disk
├─vda1                                      253:1    0  142M  0 part
├─vda2                                      253:2    0  244M  0 part
│ └─eswap-vdb2                              252:1    0  243M  0 crypt [SWAP]
└─vda3                                      253:3    0  7.3G  0 part
  └─luks-1d955539-b532-4bcb-870f-37a0c0992696
                                            252:0    0  7.3G  0 crypt /
```

Previously it would have been eswap<SIZE>, so this naming is probably better and still unique.

Reply:
cool.. Thanks for the info.. 
Curious..shouldn't it be "eswap-vd**a**2"?

Reply:
The name will be mapped on the image that is running the installer; in this case, vda would have been the source media, and vdb would be the target. It just need to be unique.